style.css

```diff
@@ -0,0 +1,187 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Indie Flower', cursive;
+  background: #F07162;
+}
+
+h1 {
+  font-family: 'Quicksand', sans-serif;
+  font-weight: bold;
+  font-size: 44px;
+  line-height: 34px;
+  color: #fff;
+  text-align: center;
+}
+
+h2 {
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 34px;
+  color: #fff;
+  text-align: center;
+  margin-bottom: 36px;
+}
+
+p {
+  font-size: 25px;
+  font-weight: 400;
+  line-height: 28px;
+  margin: 0;
+}
+
+input {
+  box-sizing: border-box;
+  border: none;
+  border-radius: 4px 0 0 4px;
+  background: #e5e9ff;
+  color: #F07162;
+  padding: 16px;
+  font-size: 16px;
+  font-family: 'Indie Flower';
+  font-weight: 600;
+  line-height: 26px;
+  flex: 1;
+}
+
+main {
+  margin: 0 auto;
+  width: 100%;
+  max-width: 1700px;
+  height: 800px;
```

Comment:
I would suggest decreasing the height since your input-wrapper is below the view.

Reply:
Tried out `vh` (view height) before?
`height: 85vh;`